addons/sourcemod/scripting/quasemago/csgo_weaponstickers/menus.inc

```diff
@@ -63,11 +58,11 @@ void ShowWeaponStickersMenu(int client, const char[] search = "")
 		
 		if (g_PlayerWeapon[client][index].m_sticker[i] != 0)
 		{
-			AddMenuItemFormat(menu, slot, _, "Slot %i\n  -> %s.", i, g_Sticker[eItems_GetStickerNumByDefIndex(g_PlayerWeapon[client][index].m_sticker[i])].m_displayName);
+			AddMenuItemFormat(menu, slot, _, "Слот %i\n  -> %s.", i, g_Sticker[eItems_GetStickerNumByDefIndex(g_PlayerWeapon[client][index].m_sticker[i])].m_displayName);
```

Comment:
Note, menu was in Russian. Would translation be available in the translation file too?